portfolio/src/main/webapp/index.html

```diff
@@ -29,9 +29,11 @@ <h1>Gia-Huy's Portfolio</h1>
     </form>
     </hr>
     <ul id="quote-list"></ul>
-
+    
```

Comment:
can you remove the extra spaces in this file as well? (line 32 and 36)

Reply:
I fixed this in info window pr :)